cloud/terraform/machineactuator.go

```diff
@@ -207,6 +207,10 @@ func (tf *TerraformClient) Create(cluster *clusterv1.Cluster, machine *clusterv1
 		if err != nil {
 			return err
 		}
+		// Save the calico.yaml file.
+		if err := saveFile(podNetworkCNICalico, path.Join("/tmp", "calico.yaml"), 0644); err != nil {
+			return errors.New(fmt.Sprintf("Could not write calico.yaml %s", err))
```

Comment:
Please change the "C" in "Could" to a "c". Errors in go should be start with a lowercase character to allow for chaining several errors together: https://github.com/golang/go/wiki/Errors.